hydradx/notebooks/Omnipool/vault/ImpermanentLossAnalysis.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import os\n",
+    "import glob\n",
+    "import random\n",
+    "sys.path.append('../../..')\n",
+    "\n",
+    "from model import processing\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "from model import run\n",
+    "from model import plot_utils as pu\n",
+    "from model.amm.omnipool_amm import OmnipoolState, cash_out_omnipool, value_assets, usd_price, lrna_price\n",
+    "from model.amm.agents import Agent\n",
+    "from model.amm.trade_strategies import omnipool_arbitrage, invest_all, price_sensitive_trading\n",
+    "from model.amm.global_state import GlobalState, fluctuate_prices, historical_prices\n",
+    "\n",
+    "# same seed, same parameters = same simulation result\n",
+    "random.seed(42)\n",
+    "# price_list = processing.import_binance_prices(['BTC', 'ETH', 'DOT'], start_date='Jan 1 2023', days = 120)\n",
+    "\n",
+    "assets = {\n",
+    "    'HDX': {'usd price': 0.05, 'weight': 0.7999},\n",
```

Comment:
HDX plays a special role in the Omnipool currently, with LRNA fees being directed to the HDX pool whenever imbalance = 0. I'd thus prefer HDX weight to be realistic (5% or so?) and for there to be some other generic token to make up the additional 65% of the pool.
Would this wind up being identical?

Reply:
The HDX/USD price shouldn't change in this notebook, right? Is that what we see? I would expect there needs to be some trade involving HDX to make this happen, which would change its weight?

Reply:
So my question above stands... and I see that we have TKN and TKN2 in both notebooks? That's what I believe is appropriate.